boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
```

Comment:
Ok I'm fine with the position and also with the name of that macro. I will trust you :-). Well, the periph_conf is meant to abstract the RIOT-driver from hardware specifics in general. In most cases that contains board specific things like pin mappings, but could also contain microcontroller attributes. If you compare #3095, this approach might change a bit in the midterm future, but I suggest to implement to "old" or "common" structure.